src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -894,6 +894,7 @@ private static bool AddHidingAbstractDiagnostic(Symbol hidingMember, Location hi
             {
                 case Accessibility.Internal:
                 case Accessibility.Private:
+                case Accessibility.ProtectedAndInternal:
```

Comment:
The principal difference between these two categories of access levels is that these ones do not cross assembly boundaries, and the other ones do. Specifically, a `private protected` member that hides a virtual member does not prevent the hidden member from being implemented (in another assembly). But a `protected` member that hides a virtual member *does* prevent it from being implemented.

Having said that, it isn't clear to me what language specification rule justifies the diagnostics produced here. I'll file an issue.